src/logic.service.ts

```diff
@@ -121,12 +139,10 @@ const executeRequest = async (
     span?.end();
     typologyResult.result = typologyResultValue;
     typologyResult.threshold = expression?.threshold ?? 0.0;
+    typologyResult.desc = expression.desc?.length ? expression.desc : noDescription;
+    typologyResult.prcgTm = calculateDuration(startHrTime, process.hrtime());
```

Comment:
Shouldn't we calculate duration as the last thing in the function?

Reply:
No I am trying to always exclude the time of successor's processing time the duration is just for TP exclusively